src/Datadog.Trace/DiagnosticListeners/AspNetCoreDiagnosticObserver.cs

```diff
@@ -39,15 +40,15 @@ internal sealed class AspNetCoreDiagnosticObserver : DiagnosticObserver
         private static readonly PropertyFetcher BeforeActionHttpContextFetcher = new PropertyFetcher("httpContext");
         private static readonly PropertyFetcher BeforeActionActionDescriptorFetcher = new PropertyFetcher("actionDescriptor");
 
-        private readonly IDatadogTracer _tracer;
+        private readonly Tracer _tracer;
         private readonly bool _isLogLevelDebugEnabled = Log.IsEnabled(LogEventLevel.Debug);
 
         public AspNetCoreDiagnosticObserver()
             : this(null)
         {
         }
 
-        public AspNetCoreDiagnosticObserver(IDatadogTracer tracer)
+        public AspNetCoreDiagnosticObserver(Tracer tracer)
```

Comment:
I'm worried about this signature change in a public .ctor

Reply:
The class is internal